### PR TITLE
feat: add helm support for governance module

### PR DESCRIPTION
**Pull request by 123liuziming**
### Describe what this PR does / why we need it
Add helm support for governance module!
### Does this pull request fix one issue?
#3244 
### Special notes for reviews
Don't merge it at once, I will check it further next week when we the Spring Cloud Alibaba's test k8s cluster is expired.

### CODE REVIEW

**Comment by 123liuziming** [comment]
I have tested it on the new environment. It seems ok.

File: spring-cloud-alibaba-examples/governance-example/helm-chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  repository: registry.cn-shenzhen.aliyuncs.com/sca-demo/default-provider-version-example
+  tag: develop
+image_provider_version:
+  repository: registry.cn-shenzhen.aliyuncs.com/sca-demo/provider-version-example
+  tag: develop
+image_istio_consumer:
+  repository: registry.cn-shenzhen.aliyuncs.com/sca-demo/istio-consumer-example
+  tag: develop
+image_nacos_server:
+  repository: registry.cn-beijing.aliyuncs.com/wangtao-mse/nacos-server
```
Comment (steverao) [inline spring-cloud-alibaba-examples/governance-example/helm-chart/values.yaml]: Please use image provided by Nacos community.

File: spring-cloud-alibaba-examples/governance-example/helm-chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+image_auth_mvc:
```
Comment (steverao) [inline spring-cloud-alibaba-examples/governance-example/helm-chart/values.yaml]: Please use yaml format to define variable. Such as:
```
image:
  authWebflux:
     xx
  defaultProviderVersion:
     xx
```

Comment (123liuziming) [reply spring-cloud-alibaba-examples/governance-example/helm-chart/values.yaml]: done

**Review by steverao — APPROVED** [review]
LGTM, thanks @123liuziming

**Pull request merged**